yast2_ntp-client_support_server-bsc1074726-20180104.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": [],

```

Comment:
still, we need a property workaround here. See other needle json files as example

Reply:
Changed!

Do I need to do the same thing for the already accepted needle for SLE?